CHANGELOG.asciidoc

```diff
@@ -20,6 +20,10 @@ limitations under the License.
 
 image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/images/gremlin-sleeping-beauty.png[width=185]
 
+==== Improvements
+
+* TINKERPOP-2764 Adds status_message as a property on GremlinServerError
```

Comment:
Another small change is required here. Instead of adding to the Improvements section here (which we populate at the time of the release with JIRAs), please add this line to the `=== TinkerPop 3.5.4 (Release Date: NOT OFFICIALLY RELEASED YET)` section below. 

Would you also consider changing the changelog message to "Expose error message sent by the server as a property on GremlinServerError for gremlin-python"